### PR TITLE
EUI 4002 screen width

### PR DESCRIPTION
**Pull request by ishOsw**
- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4002

### Change description ###

Changes to the Manage cases screen width

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by leepavlouhmcts — DISMISSED** [review]

File: src/app/containers/app/app.component.scss [hunk 1]
```diff
@@ -2,15 +2,19 @@ $govuk-global-styles: true;
...
 // Width of main container
-$hmcts-page-width: 1220px;
+$hmcts-page-width: 1920px;
 
 // TODO temporary solution to extend the width of the page until is addressed through govuk-frontend"
 body .govuk-width-container, #content {
-  max-width: 1220px;
+  max-width: 1920px;
+  min-width: 1020px;
+  overflow-x:hidden;
```
Comment (paul-graham) [inline src/app/containers/app/app.component.scss:11]: What is the `overflow-x: hidden` for? If we've got content exceeding the bounds that we just clip, that's not ideal...

**Review by paul-graham — CHANGES_REQUESTED** [review]
No margin when I think there should be...
Screenshot 2021-06-09 at 10 30 23

The header doesn't behave like the content (this is scrolled to the right):
Screenshot 2021-06-09 at 10 31 56

And nor does the footer:
Screenshot 2021-06-09 at 10 32 32

**Review by leepavlouhmcts — CHANGES_REQUESTED** [review]
on the cases page, the BETA text on the top left is out of alignment but is fine on the case details page. if this isnt because of this css fix then im hapy to approve

**Review by leepavlouhmcts — APPROVED** [review]

**Comment by paul-graham** [comment]
> on the cases page, the BETA text on the top left is out of alignment but is fine on the case details page. if this isnt because of this css fix then im hapy to approve

I think the issue is that the **Case list** has additional padding... but it already had that. Just looked on AAT and it's the same.

**Comment by paul-graham** [comment]
Some slightly odd behaviour when you go above 1920px...

Footer @ 1945px (looks fine up to 1920px):
Footer 1945px

Header @ 1860px:
header 1860px

Header @ 1920px:
header 1920px

Header @ 1979px:
header 1979px

Header @ 1980px:
header 1980px

Header (left) @ 1979px:
header left 1979px

Header (left) @ 1980px:
header left 1980px

**Review by paul-graham — COMMENTED** [review]
Some odd behaviour when you get above 1920px and especially above 1980px. Not sure this is a blocker, though...

**Review by paul-graham — APPROVED** [review]
The CSS looks good and, running it locally, it all seems to behave itself nicely. 🙂

**Review by leepavlouhmcts — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by leepavlouhmcts — APPROVED** [review]

**Pull request merged**